AndroidManifest.xsl

```diff
@@ -3,11 +3,11 @@
 	<xsl:param name="package" />
 	<xsl:param name="title" />
 	<xsl:param name="activity" />
-	<xsl:param name="version" />
-	<xsl:param name="versionCode" />
+	<xsl:param name="version">23</xsl:param>
+	<xsl:param name="versionCode">23</xsl:param>
 
-	<xsl:param name="minSdkVersion">8</xsl:param>
-	<xsl:param name="targetSdkVersion">14</xsl:param>
+	<xsl:param name="minSdkVersion">21</xsl:param>
+	<xsl:param name="targetSdkVersion">27</xsl:param>
```

Comment:
these will get overridden right ?

Reply:
Yes. Everything will be overridden by Gradle at the last moment before building Android app

Reply:
understood